<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <resources><include location="../../../res/extui/extui.qrc"/></resources><author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QG_BlockDialog</class>
  <widget class="QDialog" name="QG_BlockDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>253</width>
        <height>79</height>
      </rect>
    </property>
    <property name="baseSize">
      <size>
        <width>0</width>
        <height>0</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Block Settings</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>false</bool>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>6</number>
      </property>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <widget class="QLabel" name="lName">
              <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
                <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
                <string>Block Name:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer87">
              <property name="sizeHint">
                <size>
                  <width>29</width>
                  <height>21</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QLineEdit" name="leName">
              <property name="text">
                <string/>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <spacer name="spacer1">
              <property name="sizeHint">
                <size>
                  <width>79</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="bOk">
              <property name="text">
                <string>&amp;OK</string>
              </property>
              <property name="shortcut">
                <string>Alt+O</string>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="bCancel">
              <property name="text">
                <string>Cancel</string>
              </property>
              <property name="shortcut">
                <string>Esc</string>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <connections>
    <connection>
      <sender>bCancel</sender>
      <signal>clicked()</signal>
      <receiver>QG_BlockDialog</receiver>
      <slot>reject()</slot>
    </connection>
    <connection>
      <sender>bOk</sender>
      <signal>clicked()</signal>
      <receiver>QG_BlockDialog</receiver>
      <slot>validate()</slot>
    </connection>
  </connections>
</ui>
